scripts/zones/The_Garden_of_RuHmet/mobs/Ixaern_DRG.lua

```diff
@@ -6,25 +6,69 @@ local ID = require("scripts/zones/The_Garden_of_RuHmet/IDs")
 -----------------------------------
 local entity = {}
 
+entity.onMobSpawn = function(mob)
+    local mobId = mob:getID()
+    local x = mob:getXPos()
+    local y = mob:getYPos()
+    local z = mob:getZPos()
+    mob:useMobAbility(626) -- 2hr animation since wynavs aren't spawned via CallWyvern Ability
+    for i = mobId + 1, mobId + 3 do
+        local wynav = GetMobByID(i)
+        wynav:setSpawn(x + math.random(-2, 2), y, z + math.random(-2, 2))
+        wynav:spawn()
+    end
+end
+
 entity.onMobFight = function(mob, target)
-    -- Spawn the pets if they are despawned
-    -- TODO: summon animations?
     local mobId = mob:getID()
     local x = mob:getXPos()
     local y = mob:getYPos()
     local z = mob:getZPos()
+    local changeTime = mob:getLocalVar("changeTime")
+    local battleTime = mob:getBattleTime()
 
     for i = mobId + 1, mobId + 3 do
         local wynav = GetMobByID(i)
-        if not wynav:isSpawned() then
+        if
+            not wynav:isSpawned() and
+            utils.canUseAbility(mob) == true
+        then
             local repopWynavs = wynav:getLocalVar("repop") -- see Wynav script
             if mob:getBattleTime() - repopWynavs > 10 then
-                wynav:setSpawn(x + math.random(1, 5), y, z + math.random(1, 5))
+                wynav:setSpawn(x + math.random(-2, 2), y, z + math.random(-2, 2))
+                mob:useMobAbility(626) -- 2hr animation since wynavs aren't spawned via CallWyvern Ability
                 wynav:spawn()
                 wynav:updateEnmity(target)
             end
         end
     end
+
+    if
+        mob:getAnimationSub() <= 1 and
+        battleTime - changeTime > 60
+    then -- goes into bracer mode for 30 seconds, every 60 seconds.
+        mob:setAnimationSub(2) -- bracer mode
+        mob:setMod(xi.mod.DELAY, 1000) -- increase speed in bracer mode
+        mob:addMod(xi.mod.ATTP, 25)
+        mob:setLocalVar("changeTime", battleTime)
+    elseif
+        mob:getAnimationSub() == 2 and
+        battleTime - changeTime > 30
+    then
+        mob:setAnimationSub(1)
+        mob:setMod(xi.mod.DELAY, 0) -- back to normal attack speed
```

Comment:
this too

if its speed is changing, its likely some type of haste - probably ability haste